src/components/screens/Service.tsx

```diff
@@ -15,7 +15,34 @@ import { WrapperWrap } from '../Wrapper';
 import { DraggableGrid } from './DraggableGrid';
 import { ServiceTile, TServiceTile } from './ServiceTile';
 
-const { MATOMO_TRACKING } = consts;
+const { MATOMO_TRACKING, UMLAUT_REGEX } = consts;
+
+const umlautSwitcher = (text: string) => {
+  if (!text) return;
+
+  const replacedText = text.replace(UMLAUT_REGEX, (match: string) => {
+    switch (match) {
+      case 'ü':
+        return 'ue';
+      case 'ä':
+        return 'ae';
+      case 'ö':
+        return 'oe';
+      case 'Ü':
+        return 'UE';
+      case 'Ä':
+        return 'AE';
+      case 'Ö':
+        return 'OE';
+      case 'ß':
+        return 'ss';
+
+      default:
+        return match;
+    }
+  });
+  return replacedText;
```

Comment:
i thought that there could be a smarter solution and asked chatgpt3.5:

```
const umlautReplacements = {
  'ü': 'ue',
  'ä': 'ae',
  'ö': 'oe',
  'Ü': 'UE',
  'Ä': 'AE',
  'Ö': 'OE',
  'ß': 'ss'
};

const replacedText = text.replace(UMLAUT_REGEX, match => umlautReplacements[match]);
```

what do you think? does that work and if yes, how do you feel about it?

Reply:
It looks much more beautiful! 👍